tests/test_auth.py

```diff
@@ -0,0 +1,35 @@
+from httpx import Cookies
+
+from app import models
+from app.auth_manager import get_strategy
+from app.config import settings
+from app.utils.enums import RequestMethod
+from tests.utils import make_http_request
+
+
+async def test_refresh_token_handling(test_user: models.User):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Include tests for invalid refresh tokens.

Testing with invalid refresh tokens can help ensure the security measures are correctly implemented and robust.

```suggestion
async def test_refresh_token_handling_invalid_token(test_user: models.User):
    """
    Test function to verify the handling of invalid refresh tokens for a given test user.

    Args:
        test_user: The test user object for invalid token refresh testing.

    Returns:
        None
    """

    invalid_token = "thisisnotavalidtoken"

    cookies = Cookies(
        {
            settings.refresh_token_name: invalid_token,
        }
    )

    endpoint = "/api/users/me"

    res = await make_http_request(
        url=endpoint, method=RequestMethod.GET, cookies=cookies
    )
    assert res.status_code == 401
```